### DISCUSSION TITLE
Trying to do a local build of the api service and getting "sh: cross-env: not found" errors

### DISCUSSION THREAD

**Discussion by timmanik** [danny-avila/LibreChat#4466] [Troubleshooting]
I'm trying to do a local build of the api service.

Below is how I defined my `docker-compose.override.yml`

```yaml
services:
  api:
    image: librechat
    build:
      context: .
      target: node
```

The only changes I made to the repo were adding `.env`, `docker-compose.override.yml`, and `librechat.yml` (which is not even used in context in this deployment.

Below is the error I get when running `docker compose up --build`.

```bash
LibreChat         | > LibreChat@v0.7.5-rc2 backend
LibreChat         | > cross-env NODE_ENV=production node api/server/index.js
LibreChat         | 
LibreChat         | sh: cross-env: not found
```

There are no other errors aside from the one above.

I've ran `npm run update:docker` and still no luck. I've also tried to deploy while pruning all images, volumes, containers etc. And build with `--no-cache`.

Not sure what went wrong. I tried to edit the `package.json` at one point, but reverted it to the original version since I want to try to keep as minimal changes to the repo to test out the local build.

Let me know if any of you have any questions on my steps or need me to run certain commands to debug. Any help is much appreciated.

#### Additional context
The reason I am trying to do a local build is to test some modifications I made to `/api/server/services/Endpoints/custom/initializeClient.js`. However, for the deployment steps above, I did not change `/initializeClient.js` since I'm trying to figure out to the root cause of the deployment error above.

**Comment by timmanik** [comment]
Had to completely do full delete and re-install of docker, docker compose, etc. No more `cross-env: not found` errors.

**Comment by berg-wang** [comment]
i met the same problem, have you solved the problem?
